README.md

```diff
@@ -1,4 +1,4 @@
-[![CircleCI](https://dl.circleci.com/status-badge/img/gh/mozilla/bigquery-etl/tree/main.svg?style=svg)](https://dl.circleci.com/status-badge/redirect/gh/mozilla/bigquery-etl/tree/main)
+[![CircleCI](https://dl.circleci.com/status-badge/img/gh/mozilla/bigquery-etl/tree/main.svg?style=svg&circle-token=1df4cefd991043d7d3f13243ea80f38e7aa18341)](https://dl.circleci.com/status-badge/redirect/gh/mozilla/bigquery-etl/tree/main)
```

Comment:
Surprised we need to add the token on a public repo
```suggestion
[![mozilla](https://dl.circleci.com/status-badge/img/gh/mozilla/bigquery-etl/tree/main.svg?style=svg&circle-token=1df4cefd991043d7d3f13243ea80f38e7aa18341)](https://dl.circleci.com/status-badge/redirect/gh/mozilla/bigquery-etl/tree/main)
```

Reply:
hmm, that section (`![$alt-text]`) represents the alt-text, which may be confusing if it says "mozilla" (this link is a direct copy from circleci). I'm going to leave it as `CircleCI` for now

Reply:
I might be looking at different docs but here https://circleci.com/docs/status-badges/#generating-a-status-badge it says `ORG_NAME`: `[![<ORG_NAME>](https://circleci.com/<VCS>/<ORG_NAME>/<PROJECT_NAME>.svg?style=svg)](<LINK>)`

In the lookml-generator [README](https://github.com/mozilla/lookml-generator/blob/2d15382425553539814464390410a9fe97d1b694/README.md?plain=1#L2) we use `mozilla`.